tslint.json

```diff
@@ -37,8 +37,9 @@
       "check-module"
     ],
     "no-implicit-dependencies": [
-      true,
-      "dev"
+      false,
```

Comment:
@zFernand0 did you want to can this to `true` still?

Reply:
My apologies. Thought I already had it to true.